www/src/views/builder/index.ejs

```diff
@@ -321,23 +321,36 @@
 											<option value="3">Shield slot only</option>
 										</select>
 									</div>
+                                </div>
+								<div class="col-auto" ng-if="currentItem.slot == 13 || currentItem.slot == 2">
```

Comment:
I'm confused, according to the javascript, runecrafting happens for slots 3, 4, 14, and 15. Why is this `ng-if` set to slots 13 and 2?

Reply:
The two neck slots are 2 and the two wrist slots are 13 but when it comes to the item index its 3, 4, 14 and 15.

Reply:
gotcha